app/src/main/java/ch/epfl/sdp/peakar/user/challenge/goal/RemotePointsChallenge.java

```diff
@@ -30,13 +32,20 @@
     /**
      * Constructor for a new concrete points challenge. This constructor needs to be called to retrieve an existing challenge, not to create a new one.
      * @param id unique identifier of the challenge.
+     * @param founderID Id of the founder
+     * @param challengeName challenge name.
      * @param users users who joined the challenge.
-     * @param awardPoints points that will be awarded to the winner.
+     * @param userIDPointsMap current user ranking
+     * @param userIDUsernameMap map between user ID and user names
      */
-    public RemotePointsChallenge(String id, List<String> users, long awardPoints, int status,
+    public RemotePointsChallenge(String id, String founderID, Uri founderUri , String challengeName, List<String> users, int status,
                                  LocalDateTime creationDateTime, int durationInDays,
-                                 LocalDateTime startDateTime, LocalDateTime finishDateTime) {
-        super(id, users, awardPoints, status, creationDateTime, durationInDays, startDateTime, finishDateTime);
+                                 LocalDateTime startDateTime, LocalDateTime finishDateTime,
+                                 @Nullable HashMap<String,Integer> userIDPointsMap,
+                                 @Nullable HashMap<String,String> userIDUsernameMap) {
```

Comment:
Same as in the last comment (for example, _status_).

Reply:
Resolved